datasets/google_wellformed_query/README.md

```diff
@@ -0,0 +1,48 @@
+---
+task_categories:
+- text-scoring
+task_ids:
+- other
+multilinguality:
+- translation
+languages:
+- en
+annotations_creators:
+- crowdsourced
+source_datasets:
+- extended
+size_categories:
+- 10K<n<100K
+licenses:
+- CC-BY-SA-4.0
+---
+
+# Dataset Card for PIB
+
+## Table of Contents
+- [Dataset Card for PIB](#dataset-card-for-pib)
+  - [Table of Contents](#table-of-contents)
+    - [Dataset Description](#dataset-description)
+    - [Dataset Summary](#dataset-summary)
+    - [Citation Information](#citation-information)
+
+### Dataset Description
+
+- **Homepage:** https://github.com/google-research-datasets/query-wellformedness
+- **Paper:** https://arxiv.org/abs/1808.09419
+
+### Dataset Summary
+
+Google's query wellformedness dataset was created by crowdsourcing well-formedness annotations for 25,100 queries from the Paralex corpus. Every query was annotated by five raters each with 1/0 rating of whether or not the query is well-formed. 
+
+
+### Citation Information
+
+```
+@InProceedings{FaruquiDas2018,
+  title = {{Identifying Well-formed Natural Language Questions}},
+  author = {Faruqui, Manaal and Das, Dipanjan},
+  booktitle = {Proc. of EMNLP},
+  year = {2018}
+}
+```
```

Comment:
looks like you removed some paragraph titles of the dataset card

Could you add them back ?
You can leave their content as `[More Information Needed]`